src/CsvReader.php

```diff
@@ -178,7 +102,6 @@ protected function detectLineBreak()
     {
         rewind($this->getFilePointer());
         $sample = fread($this->getFilePointer(), 10000);
-        rewind($this->getFilePointer());
```

Comment:
Nespoléhal bych na to, že to nikdo nezavolá jen proto, že je to protected. Ta třída jde extendovat a budou z toho těžko odhalitelné chyby. Nebo co ji dát final?

Reply:
tomu nerozumim, vadi ti, ze ma detectLineBreak side effect? no a co? nejmenuje se to getLineBreak (srovnej se soucasnou verzi, kde getLineBreak ma sideeffect), stejne tak ma openCsvfile a closeCsvFile maji side effect, pokud si tu tridu nekdo extenduje a zavola to nekde uprostred cteni/zapisu, tak je proste blbej

nevidim smysl v tom snazit se tomu nejak zabranit

Reply:
ok